docker/Dockerfile.arm64

```diff
@@ -1,5 +1,5 @@
-FROM alpine:3.18 AS builder
-RUN apk add --no-cache 'crystal=1.8.2-r0' shards sqlite-static yaml-static yaml-dev libxml2-static zlib-static openssl-libs-static openssl-dev musl-dev xz-static
+FROM alpine:3.19 AS builder
+RUN apk add --no-cache 'crystal=1.10.1-r0' shards sqlite-static yaml-static yaml-dev libxml2-static zlib-static openssl-libs-static openssl-dev musl-dev xz-static
```

Comment:
Is crystal 1.12 not supported on Alpine ARM yet?

Reply:
Not on 3.19, no. It's on Edge, but the container would break almost as soon as a new Crystal release is available.